### ISSUE TITLE
strange loss

### ISSUE THREAD

**Issue by alizezhu** [MIC-DKFZ/nnUNet#197]
Hi Fabian,
I have trained a network which works very well, and when I looked at the loss curve, I think it is still converging when the epoch is 1000. So I manually changed the parameter of max number of epochs and start training it again with the -c option. However I found that the loss goes up again (it does not follow the trend at 1000)

It is the same for two folds, see the figures below. For both there is a transition at the 1000 epoch. 

![image](https://user-images.githubusercontent.com/8353480/79641095-9a79b880-8195-11ea-9f19-7e691841cc48.png)

![image](https://user-images.githubusercontent.com/8353480/79641101-a2d1f380-8195-11ea-878f-aaeb3d52ca07.png)

I cannot figure out why, something related to the learning rate?
Could you give me some hint? Thanks!

**Comment by FabianIsensee** [comment]
Hi,
the learning rate is tied to the progress in terms of epochs. At epoch 1000/1000 it will be almost zero. If you then increase the number of epochs to XXXX it will jump back up again creating this loss in accuracy. You need to overwrite the learning rate schedule for this to work properly or train with XXXX to begin with.  
These dips in the training are unusual. My best guess is that the momentum term we have as default could be too high. Please try 0.98 or even 0.95.
Best,
Fabian

**Comment by FabianIsensee** [comment]
(misclick)

**Comment by FabianIsensee** [comment]
Due to the learning rate schedule it may look like the network is not finished training when it actually is. From what your plots look like you do not need to train longer at all. BUt you should address this momentum issue

**Comment by alizezhu** [comment]
Hi Fabian, thanks for your reply. I will test the momentum. Did you remove the good old patience parameter in the new version?

**Comment by FabianIsensee** [comment]
Yeah that one got retired. The new schedule performs better, especially on more difficult datasets
